changelog/1.25.0.md

```diff
@@ -3,6 +3,13 @@ title: "1.25.0"
 description: "Released on 11/17/2021"
 ---
 
+> The final patch release of Kubernetes 1.19 was published on 28 October 2021.
+> As such, Coder v1.26 (and later) will require the use of Kubernetes 1.20 or
+> later. See Coder's [version support policy] for more information.
+
+[version support policy]:
+  ../setup/Kubernetes/index.md#supported-Kubernetes-versions
```

Comment:
hmm something changes kubernetes -> Kubernetes, but that technically breaks links since URLs are case sensitive :frowning: 

is there some way to suppress that behavior? maybe turning lint off for this line?